content/rooms/v6.md

```diff
@@ -40,11 +40,9 @@ in [room version 5](/rooms/v5).
 
 ### Authorization rules
 
-`m.room.redaction` events are not explicitly part of the auth rules.
-They are still subject to the minimum power level rules, but should always
-fall into "10. Otherwise, allow". Instead of being authorized at the time
-of receipt, they are authorized at a later stage: see the
-[Handling Redactions](#handling-redactions) section below for more information.
+While they are still subject to them, there are no auth rules specifically for
+`m.room.redaction` events. The actual redaction of content is authorized at a
+later stage: see the [Redactions](#redactions) section below for more information.
```

Comment:
I find this use of "they" and "them" to mean different things rather confusing. I think we can do better.

Also, you've changed the link, to refer to the wrong section.

```suggestion
`m.room.redaction` events are subject to auth rules in the same way as any other event.
In practice, that means they will normally be allowed by the auth rules, unless the
`m.room.power_levels` event sets a power level requirement for `m.room.redaction`
events via the `events` or `events_default` properties. In particular, the /redact
level/ is **not** considered by the auth rules.

The ability to send a redaction event does not mean that the redaction itself should
be performed. Receiving servers must perform additional checks, as described in
the [Handling Redactions](#handling-redactions) section below.
```

Suggest putting all this in a `{{% boxes/note %}}` box next to the other one, below.

Reply:
Likewise for the other room versions.

Reply:
yeah, I meant it to be italic. There are other references to "redact level" in the appendices, and they use italics.

Underscores, maybe?